network/devices/link.go

```diff
@@ -78,6 +99,33 @@ func (Link) List() (names []string, err error) {
 	return names, nil
 }
 
+func (l Link) GetStatistics() (garden.ContainerNetworkStat, error) {
+	var stat garden.ContainerNetworkStat
+
+	type netStatsPair struct {
+		data    *uint64
+		sysFile string
+	}
+
+	netStats := []netStatsPair{
```

Comment:
rather than looping over this array, why not just have a `stat(name string)` function and then this method would become :

~~~~
return garden.ContainerNetworkStat{
  RxBytes: stat("rx_bytes"),
  TxBytes: stat("tx_bytes"),
}
~~~~

I had to spend a fair while reading this method to figure out what it did, I think the above would be quite a lot easier to quickly understand